src/simple_lottery.rs

```diff
@@ -0,0 +1,185 @@
+//! ALBA Telescope with simple lottery construction given in Section 4.1.
+
+/// Setup output parameters
+#[derive(Debug, Clone)]
+pub struct LotterySetup {
+    /// Proof size (in Sp elements)
+    pub u: u64,
+    /// Lottery oracle probability
+    pub p: f64,
+}
+
+impl LotterySetup {
+    /// Setup algorithm taking as input the security parameters, the set size,
+    /// and the lower bound and returns LotterySetup.
+    pub fn new(lambda_sec: f64, lambda_rel: f64, n_p: u64, n_f: u64) -> Self {
+        let (u, mu) = Self::compute_u_mu(lambda_sec, lambda_rel, n_p, n_f);
```

Comment:
Section 4.1 introduces the parameter `mu` (the expected number of network participants) - not sure how exactly we measure expectation here. It might be helpful to clarify what `expected` exactly means - is the expectation calculated over a long period of network activity where nodes go online/offline?